shared/qt/idian/widgets/PropertiesList.cpp

```diff
@@ -0,0 +1,83 @@
+/******************************************************************************
+    Copyright (C) 2023 by Dennis Sädtler <dennis@obsproject.com>
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+******************************************************************************/
+
+#include <QStyle>
+
+#include <Idian/Row.hpp>
+
+#include <Idian/moc_PropertiesList.cpp>
+
+using idian::PropertiesList;
+
+PropertiesList::PropertiesList(QWidget *parent) : QFrame(parent)
+{
+	layout = new QVBoxLayout();
+	layout->setSpacing(0);
+	layout->setContentsMargins(0, 0, 0, 0);
+	setSizePolicy(QSizePolicy::MinimumExpanding, QSizePolicy::Minimum);
+
+	rowsList = QList<GenericRow *>();
+
+	setLayout(layout);
+}
+
+/* Note: This function takes ownership of the added widget
+ * and it may be deleted when the properties list is destroyed
+ * or the clear() method is called! */
```

Comment:
I'd prefer if C++ comment style is used here. Functionally this seems to be in line with Qt's ownership model (parents always own their children and thus parents take care of destroying them as well).